src/components/navigation/z-user-dropdown/index.tsx

```diff
@@ -165,7 +165,9 @@ export class ZUserDropdown {
                   htmlid={link.id}
                   target={link.target}
                   icon={link.icon}
-                  onZLinkClick={(e) => this.handleDropdownLinkClick(e)}
+                  onZLinkClick={(e: CustomEvent) =>
+                    this.handleDropdownLinkClick(e)
```

Comment:
this should also work
```tsx
onZLinkClick={this.emitDropdownMenuLinkClick}
```

if so, `handleDropdownLinkClick` may be deleted

Reply:
probably the `this` reference is not correct. Maybe with `bind`
```js
onZLinkClick={this.emitDropdownMenuLinkClick.bind(this)}
```

Reply:
fixed